mantle/kola/tests/misc/network.go

```diff
@@ -209,310 +197,320 @@ func NetworkInitramfsSecondBoot(c cluster.TestCluster) {
-          	case $dev_mac in
-          	  $primary_mac)
-                    default_device=$dev
-                    profile_name=$(nmcli -g GENERAL.CONNECTION dev show $dev)
-                    ;;
-          	  $secondary_mac)
-                    secondary_device=$dev
-                    secondary_profile_name=$(nmcli -g GENERAL.CONNECTION dev show $dev)
-                    ;;
-          	  *)
-                    ;;
-          	 esac
-            done
-            echo -e "default dev: $default_device ($profile_name)\nsecondary dev: $secondary_device ($secondary_profile_name)"
-
-            mac=$(sudo nmcli -g GENERAL.HWADDR dev show $default_device | sed -e 's/\\//g')
-
-            # make bridge
-            nmcli conn add type ovs-bridge conn.interface brcnv con-name ovs-bridge-brcnv
-            nmcli conn add type ovs-port conn.interface brcnv-port master brcnv
-            nmcli conn add type ovs-interface con-name ovs-brcnv-iface conn.interface brcnv master brcnv-port ipv4.method auto ipv4.dhcp-client-id "mac" connection.autoconnect no 802-3-ethernet.cloned-mac-address $mac
-
-            # make bond
-            nmcli conn add type ovs-port conn.interface bond0 master brcnv ovs-port.bond-mode balance-slb con-name ovs-slave-bond0
-            nmcli conn add type ethernet conn.interface $default_device master bond0 con-name ovs-slave-1
-            nmcli conn add type ethernet conn.interface $secondary_device master bond0 con-name ovs-slave-2
-            nmcli conn down "$profile_name" || true
-            nmcli conn mod "$profile_name" connection.autoconnect no || true
-            nmcli conn down "$secondary_profile_name" || true
-            nmcli conn mod "$secondary_profile_name" connection.autoconnect no || true
-            if ! nmcli conn up ovs-brcnv-iface; then
-          	  nmcli conn up "$profile_name" || true
-          	  nmcli conn mod "$profile_name" connection.autoconnect yes
-          	  nmcli conn up "$secondary_profile_name" || true
-          	  nmcli conn mod "$secondary_profile_name" connection.autoconnect yes
-          	  nmcli conn delete $(nmcli c show |grep ovs-cnv |awk '{print $1}') || true
-            else
-          	  nmcli conn mod ovs-brcnv-iface connection.autoconnect yes
-          	  nmcli conn up ovs-slave-2
-          	  # Remove Old NM config files and copy the new-ones
-          	  rm -rf /var/ovsbond
-          	  cp -r $NMPATH /var/ovsbond || true
-            fi
-          else
-          	echo "ovs bridge already present"
-          	for c in ovs-bridge-brcnv ovs-slave-bond0 ovs-slave-brcnv-port ovs-slave-1 ovs-slave-2 ovs-brcnv-iface; do nmcli c up $c; done
-          fi
-`
+	initInterfacesSystemdContents = `
+	[Unit]
+	Before=kubelet.service
+	After=NetworkManager.service capture-macs.service
+	[Service]
+	Type=oneshot
+	ExecStart=/bin/sh /var/init-interfaces.sh
+	[Install]
+	WantedBy=multi-user.target
+	`
```

Comment:
The latest version of this PR doesn't pass locally for me. At the very least these tabs need to be dropped. And similarly in the next unit below.

I think there's something else wrong with string substitution into the Ignition config that needs investigating.

Reply:
Will take a look